index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100&family=Meera+Inimai&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="header-section">
+        <a href="/" class="title">Nelson Ongiti</a>
+        <nav class="menu">
+            <a href="#" class="toggle-icon">
+                <i class="fa fa-bars"></i>
+            </a>
+        </nav>
+    </header>
+
+    <section class="hero-section">
+        <div class="hero-title">
+            <h1>Hey There. <br />I’m Nelson</h1>
+            <h3>I’am a Software Developer</h3>
+        </div>
+      
+      <p class="hero-desc">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="hero-social-links">
+        <li><img src="./images/Footericons/1.png" alt="" /></li>
+        <li><img src="./images/Footericons/2.png" alt="" /></li>
+        <li><img src="./images/Footericons/3.png" alt="" /></li>
+       <li><img src="./images/Footericons/4.png" alt="" /></li>
+        <li><img src="./images/Footericons/5.png" alt="" /></li>
```

Comment:
- [ ] Your portfolio is looking great, well done! However, as the previous reviewer said, it would be great to add an `a` tag to every icon in order to make them clickable.

Reply:
I did that and webhint threw errors that the a tag is not allowed on the links and i had no other ways to bypass the errors

Reply:
kindly advice